workflow-service/README.md

```diff
@@ -22,10 +22,19 @@ test the service. It should be removed once actual Tasks and WorkFlows start get
 To start the application run the following from the root folder of 'workflow-service'.
 
 ```shell
-
-java -Dspring.profiles.active=local -jar target/workflow-service-0.0.1-SNAPSHOT.jar
-
+VERSION=1.0.3-SNAPSHOT
```

Comment:
Do you want use to change this version every time we release?

Reply:
Yes, we need too

Reply:
> Do you want use to change this version every time we release?

TBH, no - I'd rather have a template of this script and generate it based on the version each time. Let maintenance burden on us.